pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[build-system]
+requires = ["pdm-pep517>=1.0"]
+build-backend = "pdm.pep517.api"
+
+[project]
+name = "dicom-anonymizer"
+version = '1.0.11'
```

Comment:
Is `1.0.11` the latest? Maybe we should be at `1.0.7` or something similar, relevant to this fork.